dotnet/src/Connectors/Connectors.OpenAI/OpenAIPromptExecutionSettings.cs

```diff
@@ -254,6 +254,39 @@ public int ResultsPerPrompt
         }
     }
 
+    /// <summary>
+    /// Whether to return log probabilities of the output tokens or not.
+    /// If true, returns the log probabilities of each output token returned in the `content` of `message`.
+    /// </summary>
+    [Experimental("SKEXP0010")]
+    [JsonPropertyName("logprobs")]
+    public bool? EnableLogProbabilities
```

Comment:
Elsewhere we match the property name with the JSON property name. Why are we changing the pattern for these two properties?

I'd expect them to be `Logprobs` and `TopLogprobs`

Reply:
Renamed to `Logprobs` and `TopLogprobs`.